docs/rst/notes/forthcoming_version.rst

```diff
@@ -1,7 +1,41 @@
 
 .. _forthcoming_version:
 
+This release includes the following **mayor changes**:
+
+* New DDS Router library that provides the DDS Router features through a C++ API.
+* Division of DDS Router application into several packages.
+
+  - `ddsrouter_event`: C++ library which implements System Operating (SO)-dependent signal handlers.
+  - `ddsrouter_utils`:  C++ library which implements various utility functions.
+  - `ddsrouter_core`: C++ library which implements the DDS Router operation and exports the DDS Router C++ API.
+  - `ddsrouter_yaml`:  C++ library to parse the DDS Router *yaml* configuration files.
+  - `ddsrouter_tool`: DDS Router end-user application.
+
+* Build internal topic tracks via the `builtin-topics` configuration.
+
+This release includes the following **Routing features**:
+
+* Apply DDS Topic filter rules using the `allowlist` and `blocklist` lists.
```

Comment:
Dynamic module is not included in this release. I do not understand what do you mean here.

Reply:
Include in PR #154